cli/cmd/check.go

```diff
@@ -182,7 +187,7 @@ func configureAndRunChecks(wout io.Writer, werr io.Writer, stage string, options
 		APIAddr:               apiAddr,
 		VersionOverride:       options.versionOverride,
 		RetryDeadline:         time.Now().Add(options.wait),
-		NoInitContainer:       options.cniEnabled,
+		NoInitContainer:       cniEnabled,
```

Comment:
Not related to this PR, but I'll just note that this being a negative is really confusing to me, especially when we get into the double negatives.